app/design/frontend/Magento/luma/web/css/source/_buttons.less

```diff
@@ -17,8 +17,6 @@
 
     button {
         &:not(.primary) {
-            .lib-css(box-shadow, @button__shadow);
-
             &:active {
```

Comment:
@edenduong this line apply box shadow for every button doesn't have class primary. In luma theme you can see every button will have white space on top
After this change can affect to all button. I'm not sure about reason of this rule here